osp/core/cuds.py

```diff
@@ -816,8 +802,7 @@ def _load_cuds_objects(self, uids):
         Yields:
             Cuds: The loaded cuds_objects
         """
-        without_none = [uid for uid in uids
-                        if uid is not None]
+        without_none = filter(lambda x: x is not None, uids)
```

Comment:
```py
without_none = filter(None, uids)
```

Reply:
```
In [1]: import uuid

In [2]: uid = uuid.UUID(int=0)

In [3]: bool(uid)
Out[3]: True

```

Ok it's fine to do what you suggest. Thanks!